### ISSUE TITLE
cmake install script should install RPR libraries to $USD_ROOT/lib

### ISSUE THREAD

**Issue by bsavery** [GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD#27]
Right now we have to manually copy libraries to USD_ROOT/lib.  It would be nice to install them with the cmake install

**Comment by hshakula** [comment]
Hi, you can use -DCMAKE_INSTALL_PREFIX=$USD_ROOT as cmake configuration argument for this purpose

**Comment by bsavery** [comment]
Hmm, that didn't work last time I think?  Let me try again.  Might have missed the argument.

**Comment by bsavery** [comment]
Nope, libTahoe64.dylib is not installed with "make install" to USD_ROOT/lib.  It is necessary to make the context.  

I have set -DCMAKE_INSTALL_PREFIX=/data/USD
